getting_started_with_acceptance_tests.md

```diff
@@ -0,0 +1,80 @@
+
+- The onboarding of a single person to our software engineering team took more time than we desired. This was mainly being cause due to legacy code and the bad documentation of features and processes.
+- Software engineers were not able to determine if a specific defect fix or feature broke older functionality.
+- Slow production line. We needed to spend a lot of time manually testing if new functionality behaved as expected and did not break anything else.
+
+As a result of this, we decided we would start off by changing our development workflow to a more collaborative and documented process, where changes to the codebase followed a specific set of actions. These actions are:
+  - All new features must include tests (bonus points for using TDD).
+  - Bug fixes must include a corresponding regression test.
+  - One test must be added to the existing codebase every week.
+
+> You can read more about them in this blog post: [Refactoring Legacy Code](https://github.com/ubiqua/engblog/blob/master/refactoring_legacy_code.md).
+
+One of the tools we chose for our plan of action is [Cucumber](https://cucumber.io/), and in this blogpost I will guide you through the basics and good practices of writing acceptance tests with it.
+
+## Acceptance Tests
+An acceptance test demonstrates what a feature (or system) needs to do in order for the stakeholders to find it acceptable. At Ubiqua, we like to think of testing in the following way:
+
+- Acceptance tests: build the right thing.
+- Unit tests: build the thing right.
+
+Cucumber allows us to write acceptance tests by specifying a concrete number of executable examples (build the right thing) of how the system should behave under certain circumstances. Writing down these examples encourages a team to think about the different use-cases that a feature needs to serve which in turn assists in revealing insights about other feature use-cases that were unknown at the beginning of the iteration. Such practices help to avoid a [solution looking for a problem](http://xyproblem.info/), and consecutively boost the team's productivity.
+
+## Overview of a Cucumber Feature
+A Cucumber feature is a set of examples of how the system should behave under certain circumstances. Each feature is made of a number of examples called ``Scenarios``. Scenarios must be independent and usually follow this pattern:
+
+- Context: Put the system in a particular state (``Given``).
+- Action: Do something with the system (``When``).
+- Expectation: Examine the new state is correct (``Then``).
+
+Features in Cucumber are written in ``Gherkin``. Gherkin is a language used to facilitate communication with stakeholders to illustrate what the software should do. The greatest advantage of using Gherkin is that it is designed for human readability and thus helps developers to interact with stakeholders and create documentation both at the same time.
```

Comment:
Consider moving the "both" before "to interact", ie: "helps developers both to interact ... and create ... "